gdal/doc/source/development/rfc/rfc83_use_of_project_sponsorship.rst

```diff
@@ -0,0 +1,228 @@
+.. _rfc-83:
+
+=============================================================
+RFC 83: guidelines for the use of GDAL project sponsorship
+=============================================================
+
+============== =============================================
+Author:        Even Rouault (with content from RFC 9)
+Contact:       even.rouault @ spatialys.com
+Started:       2021-May-19
+Status:        Development
+============== =============================================
+
+Summary
+-------
+
+Per :ref:`rfc-80`, the GDAL project benefits from a multi-year budget coming
+from sponsorship. This document formalizes guidelines for how that budget will
+be spent by the project and supersedes :ref:`rfc-9`
+
+There are two different type of tasks that will be funded:
+
+- Day-to-day maintenance tasks. They will be filled by several co-maintainers.
+- Tasks to achieve a specific development
+
+Day-to-day maintenance tasks
+----------------------------
+
+Scope
++++++
+
+- Bug tracker maintenance:
+
+    * Making sure that the reporter provides all details needed for
+      reproducibility.
+    * Closing reports that don't respect the guidelines, or are invalid.
+    * Tagging the issue with appropriate labels and milestones.
+    * When a fix is relevant, addressing the issue with a pull request.
+    * Adding / enhancing tests in the regression test suite should be done as
+      much as practical
+    * Tasks that are expected, or are found, to take more than a given time
+      should be confirmed with the supervisor.
+
+- Continuous integration maintenance and improvements: the project has several
+  continuous integration configurations. Some of them break regularly due to
+  external causes (network resources no longer available, change in
+  dependencies, etc.). The co-maintainer will take the necessary actions to
+  keep them in working state.
+- Review of code and documentation contributions. Such contributions are made
+  currently through GitHub pull requests. The co-maintainer will review the
+  contribution and guide the proponent through the steps to make it accepted:
+  identifying issues reported by continuous integration, asking for new tests
+  to be developed whenever possible, etc. When the contributor is not in capacity
+  to polish his/her contribution, the co-maintainer may do it him/herself.
```

Comment:
it could be singular plural "their" instead of "his/her" https://www.grammar-monster.com/lessons/his_her_their_they_singular_plural.htm but how does it turn with him/herself?

Reply:
`themself`